--- a/src/libANGLE/renderer/d3d/d3d11/converged/CompositorNativeWindow11.h
+++ b/src/libANGLE/renderer/d3d/d3d11/converged/CompositorNativeWindow11.h
@@ -12,7 +12,7 @@
 
 #include "libANGLE/renderer/d3d/d3d11/NativeWindow11.h"
 
-#include <DispatcherQueue.h>
+#include <dispatcherqueue.h>
 #include <windows.foundation.metadata.h>
 #include <windows.ui.composition.h>
 #include <windows.ui.composition.interop.h>
